### ISSUE TITLE
RE: GPS and Software for the BBGW/Seth

### ISSUE THREAD

**Issue by silver2row** [Pillar1989/upm#4]
Hello,

Do I need to use the Grove Cape for this example of grovegprs.py?

Seth

P.S. I am asking because the software works but when I plug in the GPS sensor to the BBGW, I get error codes when attempting to run the software in PuTTY.

**Comment by Pillar1989** [comment]
What's you error code ? which Grove GPRS ?

**Comment by silver2row** [comment]
Hello...I ran the software again and it ran smoothly. I am sorry to bother you.

Seth

**Comment by silver2row** [comment]
Hello Again,

Here is the error code: **Error: Cannot find module 'jsupm_grovegprs'**

I have the Grove v1.2. I am trying to run this software I found on UPM in the examples for Javascript.

Seth

P.S. If you have any questions, please ask away.

**Comment by silver2row** [comment]
Hello...

I figured out that I must build/compile the UPM repository before getting into using the software provided by SeeedStudio on UPM's examples.

Here are some other issues I came across while building:

Install the project...
-- Install configuration: ""
-- Installing: /usr/local/include/mraa.h
CMake Error at src/cmake_install.cmake:36 (file):
  file INSTALL cannot copy file "/home/debian/mraa/api/mraa.h" to
  "/usr/local/include/mraa.h".
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)


Makefile:72: recipe for target 'install' failed
make: *** [install] Error 1

I hope this makes some sense. I am sort of stuck right now.

Seth
